litellm/llms/gemini.py

```diff
@@ -160,7 +163,7 @@ def completion(
         input=prompt,
         api_key="",
         original_response=response,
-        additional_args={"complete_input_dict": {}},
+        additional_args={"complete_input_dict": {}, "safety_settings": safety_settings},
```

Comment:
I don't know the best practices around litellm logging to know if this is a good idea, please advice

Reply:
that's fine